### ISSUE TITLE
Error: @prisma/client did not initialize yet

### ISSUE THREAD

**Issue by Jun0vn** [prisma/prisma#20484]
### Bug description

```
01.08 18:37:26 [Bot] 01/08/23 19:37:26  [[ERROR]] Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
01.08 18:37:26 [Bot] In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
01.08 18:37:26 [Bot] at new PrismaClient (/node_modules/.prisma/client/index.js:5:11)
01.08 18:37:26 [Bot] at Client.login (/src/client.js:59:17)
01.08 18:37:26 [Bot] at Object.<anonymous> (/src/index.js:73:8)
01.08 18:37:26 [Bot] at Module._compile (node:internal/modules/cjs/loader:1267:14)
01.08 18:37:26 [Bot] at Module._extensions..js (node:internal/modules/cjs/loader:1321:10)
01.08 18:37:26 [Bot] at Module.load (node:internal/modules/cjs/loader:1125:32)
01.08 18:37:26 [Bot] at Module._load (node:internal/modules/cjs/loader:965:12)
01.08 18:37:26 [Bot] at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
01.08 18:37:26 [Bot] at node:internal/main/run_main_module:23:47
```

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
Did you run `prisma generate` before this happened?
What version of Prisma are you running?
Can you please fill in the issue template with the information it asks for? Thanks.

**Comment by Jun0vn** [comment]
i have run prisma generate but got this error even though i have fully installed prisma/prisma client
`PS C:\Users\Juno\Desktop\pebblehost-547279-jvycah\MineTNT Ticket> prisma generate
prisma : The term 'prisma' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is 
correct and try again.
At line:1 char:1
+ prisma generate
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (prisma:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`

**Comment by janpio** [comment]
That looks like you do not have a global installation, so you probably need to use `npx prisma generate` to run this command (or whatever equivalent tool to `npx` your pacakge manager uses, if you do not use `npm`).

What version of PRisma are you using? `npx prisma -v` should output that (or you can look it up in your `package.json`).

**Comment by SevInf** [comment]
Hey @Jun0vn  ,

Did you get a chance to have a look at @janpio's last comment? It'd be helpful for us to move your issue forward.

Without further information, we'll have to close this issue. Thank you 🙏

**Comment by jkomyno** [comment]
Hey @Jun0vn, could you please reply to the comments above? Thank you.

**Comment by Erikdegroot89** [comment]
Hi, i ran into the same issue version 5. This info might provide insight:
I was getting this upgrading from `4.16.2` to `5.2.0`. When running `npx prisma generate`, i got the same error:
```
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
    at new PrismaClient (/xyz/node_modules/.prisma/client/index.js:5:11)
    at new PrismaService (/xyz/fr-platform/node_modules/nestjs-prisma/lib/prisma.service.ts:26:5)
    at Injector.instantiateClass (/xyz/node_modules/@nestjs/core/injector/[inje](injector.js:365:19)
```

Turns out when i ran `npm list @prisma/client` that the dependency i used to wrap the client (`nestjs-prisma@0.22.0` ) was still using `4.16.2`. Dropping the node modules and package lock and reinstalling fixed the issue.

So in short; this was probably caused by the `@prisma/client` and `prisma` not having the same version. ( in this case `5.2.0` )

**Comment by miguelff** [comment]
@Erikdegroot89 provided a potential explanation and fix to the problem subject of this issue in https://github.com/prisma/prisma/issues/20484#issuecomment-1697738713

Given the lack of response from the original report we are closing this.
